Lib/fontbakery/profiles/universal.py

```diff
@@ -287,6 +288,53 @@ def com_google_fonts_check_family_single_directory(fonts):
         )
 
 
+@check(
+    id="com.google.fonts/check/caps_vertically_centered",
+    proposal="https://github.com/googlefonts/fontbakery/issues/4139",
+    rationale="""
+        If possible, the uppercase glyphs should be vertically centered in the em box.
+        This allows for easier centering of text in buttons and grid systems.
+        There is a detailed twitter thread about this subject here:
+        https://twitter.com/romanshamin_en/status/1562801657691672576
```

Comment:
How do you handle multi script families say a Latin + Arabic font? The Arabic may have taller and deeper glyphs so do you try and set the metrics so the Arabic is centred or the Latin? Either way, one of the scripts may appear misaligned.

Reply:
There should be a note in the warning that is mainly a suggestion for fonts with Latin, Greek or Cyrillic as "main" script.

Reply:
I added a note about Arabic and other non-latin scripts to the rationale.